tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -112,5 +119,24 @@ sub test_flags() {
     return {fatal => 1};
 }
 
+sub post_fail_hook() {
+    my $self = shift;
+
+    if (get_var("DESKTOP") =~ /textmode|minimalx/) {
+        send_key "ctrl-alt-f1";
```

Comment:
because in textmode, the test was already in tty2 which actived by select_console 'root_console', if call select_console again, it would stay in the same tty. 
I wonder to know there is a better way to switch from tty to another, such like tty2 to ttyx?